src/Main.java

```diff
@@ -1,5 +1,64 @@
+import car.Car;
+import car.CarOwner;
+import carwash.*;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        CheapWash cheapWash = new CheapWash();
+        RegularWash regularWash = new RegularWash();
+        PremiumWash premiumWash = new PremiumWash();
```

Comment:
Use common interface here `WashStrategy` and update it via setter. Don't create multiple instances of service